content/methods/discover/system-map.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: System map
-description: A diagram that shows how different factors and forces affect a given system. Unlike [journey maps](https://guides.18f.gov/methods/decide/journey-mapping/) or [service blueprints](https://guides.18f.gov/methods/decide/service-blueprint/), which visualize specific services.
+description: A diagram that shows how different factors and forces affect a given system. Unlike journey maps or service blueprints, which visualize specific services.
```

Comment:
The second sentence here is a fragment. (This is replicated in `method.what`.)

Reply:
Fixed!